agent/command.go

```diff
@@ -139,7 +139,9 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				}
 			}
 		case <-ctx.Done():
-			tc.logger.Task().Errorf("Command stopped early: %s", ctx.Err())
+			if ctx.Err() == context.DeadlineExceeded {
```

Comment:
We should still log something (maybe just the original message) in the else case